src/masterless.py

```diff
@@ -0,0 +1,60 @@
+from fabric.api import local, task
+from decorators import requires_project, requires_aws_stack, echo_output
+from buildercore import bootstrap, config, core
+from buildercore.utils import ensure
+import logging
+from functools import wraps
+LOG = logging.getLogger(__name__)
+
+def requires_master_server_access(fn):
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        import master
+        ensure(master.server_access(), "this command requires access to the master server. you don't have it.")
+        return fn(*args, **kwargs)
+    return wrapper
+
+@task
+@requires_project
+@requires_master_server_access
+def launch(pname, instance_id=None):
+    import cfn
+    cfn.launch(pname, instance_id, 'masterless')
+    # opportunity to do post-launch things here
```

Comment:
you asked in the comments above about changing the name of the alt-aws config from 'masterless' to 'masterless-sample' - which is fine we can do that - but if we have a 'masterless' alt config like we have an 'end2end' or 'prod' config, then we can target those masterless instances in here.

although saying that, the only thing making a instance masterless right now is the `aws.ec2.masterless: True` key. I'd be wary of introducing anything else in `launch` that couldn't be done with regular `cfn.launch`. I like the fact we could have a `masterless-test` altconfig later on.